code/_onclick/click_override.dm

```diff
@@ -74,8 +74,9 @@
 		if(isliving(target_atom))
 			var/mob/living/L = target_atom
 			if(user.a_intent == INTENT_DISARM)
-				add_attack_logs(user, L, "shocked and weakened with power gloves")
-				L.Weaken(6 SECONDS)
+				add_attack_logs(user, L, "shocked and stunned with power gloves.")
```

Comment:
Since they aren't strictly stunned, maybe only worth noting that they were strictly stunned if it'd put them over 120stam? Otherwise, might want to reword

Reply:
could just do "shocked with power gloves"

Reply:
agree with sirryan, tidy that and it's good to go